client/shared/navbar/_NavbarStyle.scss

```diff
@@ -93,6 +93,20 @@
   }
 }
 
+/* Logout Button */
+.navbar__logout{
+  float: right;
+  @include rem(margin, 9px 0px 0px 5px);
+}
+
+.navbar__logout a {
```

Comment:
@whit537: I don't nest because, in many cases, it lends itself to over-specification, and can obscure opportunities to streamline. Since nested and non-nested SCSS files compile to the same minified result, I prioritize legibility above the efficiency nesting can provide. There are exceptions to the rule (media queries, pseudo-classes, etc.), but that's my general principle.

@matthewvita: looks good, but we may want to reduce its prominence on the page slightly? We don't want to hide the logout link from folks, but we don't necessarily want to highlight it, either. What about dropping it to a text link, and moving it up above the main nav. Still accessible, but less prominent. See this quick and/or dirty comp I made with dev tools:

![screen shot 2015-12-07 at 5 59 01 pm](https://cloud.githubusercontent.com/assets/2473472/11642532/3c96d882-9d0c-11e5-9722-90e8873829db.png)